tests/test_execute.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Integration tests for the ``execute`` method of LightningQubit.
+"""
+import pytest
+
+import pennylane as qml
+from pennylane import numpy as np
+
+
+@pytest.mark.parametrize("diff_method", ("param_shift", "finite_diff"))
+class TestQChem:
+    """Test tapes returning the expectation values of a Hamiltonian, with a qchem workflow."""
+
+    def test_VQE_gradients(self, diff_method, tol):
+        """Test if the VQE procedure returns the expected gradients."""
+
+        symbols = ["H", "H"]
+
+        geometry = np.array(
+            [[-0.676411907, 0.000000000, 0.000000000], [0.676411907, 0.000000000, 0.000000000]],
```

Comment:
Any reason for using this geometry for tests?

Reply:
This is a typical distance for hydrogen atoms in an H2 molecule.

Reply:
It could be any number though this is not important for the test.